Update firebase deployment document (#195)
software-platform/monorepo: docs/09_firebase_deployment.md

## Configuring Firestore database

Once you've deployed the metrics application to Firebase Hosting, you should finish configuring your Firestore database. 
The `metrics/firebase` folder contains Firestore security rules, Firestore indexes, and a `seedData` Cloud function needed to create a test data for our application. To deploy all of these components, follow the next steps: 

1. Activate the `seedData` function: go to the `metrics/firebase/functions/index.js` file and change the `const inactive = true;` to `const inactive = false;`.
2. 0pen the terminal and navigate to `metrics/firebase` folder.
3. Run the `firebase deploy` command to deploy all components.

> 4. Once command execution finished, you'll find the `seedData` function URL, used to trigger it, in the console. Save this URL somewhere - you will need it a bit later.

Reviewer: ```suggestion
4. Once command execution finished, you'll find the `seedData` function URL, that can be used to trigger function execution in the console. Save this URL somewhere - you will need it a bit later.
```

1. Go to the `metrics/firebase/functions/index.js` file and change the `inactive` constant back to `true`.
2. Redeploy this function, using the `firebase deploy --only functions` command.

## Creating a new Firebase User

Once you've finished deploying the Metrics application and created the test data, you probably want to open the application and ensure it works well, so you need to create a Firebase User to log-in to the application:

1. Go to the [Firebase Console](https://console.firebase.google.com/) and select the project, created in the previous steps.
2. Go to the `Authentication` section on the left panel and tap on the `Add user` button.
3. Provide an email and a password to create a new user and type on the `Add user` button again.
4. You should see your email with additional data appear in the list of the users' table.

With that in place, you can use your credentials, that you've used to create the user, to fill an authentication form of the web application. 

> After logged-in you should see a dashboard page with a list of test projects and their metrics, that you've created in the previous steps.

Reviewer: ```suggestion
After logging in, you should see a dashboard page with a list of test projects and their metrics if you've created them in the previous steps or no data. The app should provide an ability to switch between light/dark themes.
```

- Deploy Metrics application to Firebase Hosting.
- Deploy the Cloud Function for creating test data.

> - Deploy Firestore Security Rules.

Reviewer: ```suggestion
- Deploy Firestore Security Rules & Indexes.
```
